testing/endtoend/mainnet_e2e_test.go

```diff
@@ -10,10 +10,10 @@ import (
 
 // Run mainnet e2e config with the current release validator against latest beacon node.
 func TestEndToEnd_MainnetConfig_ValidatorAtCurrentRelease(t *testing.T) {
-	r := e2eMainnet(t, true, false, types.StartAt(version.Phase0, params.E2EMainnetTestConfig()))
+	r := e2eMainnet(t, true, false, types.StartAt(version.Phase0, params.E2EMainnetTestConfig(), false))
```

Comment:
I think adding this inline comment helps the reader understand this parameter. Ideally, all of the "true" and "false" values in this method call would have the inline so the reader doesn't have to go look it up. 

```suggestion
	r := e2eMainnet(t, true, false, types.StartAt(version.Phase0, params.E2EMainnetTestConfig(), false /*multiclient*/))
```